### PR TITLE
Add C++ tool for merging bags to fix issues with rosjava

### PR DESCRIPTION
**Pull request by oleg-alexandrov**

### CODE REVIEW

File: localization/sparse_mapping/tools/merge_bags.py [hunk 1]
```diff
@@ -1,77 +0,0 @@
...
-from rosbag import Bag
-
-def main():
-
-    parser = argparse.ArgumentParser(description='Merge one or more bag files with the possibilities of filtering topics.')
-    parser.add_argument('outputbag', help='Output bag file with topics merged.')
-    parser.add_argument('inputbag', nargs='+', help='Input bag files.')
-    parser.add_argument('-v', '--verbose', action="store_true", default=False,
-                        help='Verbose output.')
-    parser.add_argument('-t', '--topics', default="*",
```
Comment (bcoltin) [inline localization/sparse_mapping/tools/merge_bags.py:36]: The previous version had an option to filter topics, is this something we still need in the merge bags script? @rsoussan @marinagmoreira @kbrowne15

**Review by bcoltin — COMMENTED** [review]
Looks good, once the tests pass, we confirm the calibration is ok, and we wait a bit for people saying they need the filtering we can merge it.

File: astrobee/config/robots/bumble.config [hunk 2]
```diff
@@ -25,14 +25,14 @@ robot_i2c_bus = "/dev/i2c-1"
 robot_imu_drdy_pin = 4
 
 robot_geometry = {
-  hazcam_to_navcam_transform = transform(vec3(0.065116384, -0.021802268, 0.00013643523), quat4(0.0028326438, 0.026320987, 0.99964444, -0.0031901921)),
-  scicam_to_hazcam_transform = transform(vec3(0.047858317, -0.025959231, -0.089604479), quat4(-0.01481102, 0.027928636, 0.9994844, -0.0056168734)),
+  hazcam_to_navcam_transform = transform(vec3(0.070345242, -0.0088002726, -0.015396831), quat4(-0.0021496953, 0.01313305, 0.99988913, -0.0066800841)),
```
[the remark this answers is not in the document]
Comment (bcoltin) [reply astrobee/config/robots/bumble.config:28]: Never mind, I saw I missed your comment on the pull request!

File: localization/sparse_mapping/tools/merge_bags.py (hunk 1 above)
Comment (marinagmoreira) [reply localization/sparse_mapping/tools/merge_bags.py:36]: I don't need it. We can filter out topics after the bags are merged using the rosbag tool.

**Comment by marinagmoreira** [comment]
Thank you for doing this!

File: astrobee/config/robots/bumble.config (hunk 2 above)
[the remark this answers is not in the document]
Comment (oleg-alexandrov) [reply astrobee/config/robots/bumble.config:28]: Yes I meant to test them and these were very carefully tested with the latest ISS data.

**Pull request merged**